xbmc/windowing/rpi/RPIUtils.cpp

```diff
@@ -560,8 +560,8 @@ void CRPIUtils::GetSupportedModes(HDMI_RES_GROUP_T group, std::vector<RESOLUTION
         m_desktopRes = res;
 
       AddUniqueResolution(res, resolutions);
-      CLog::Log(LOGDEBUG, "EGL mode %d: %s (%.2f) %s%s:%x\n", i, res.strMode.c_str(), res.fPixelRatio,
-          tv->native ? "N" : "", tv->scan_mode ? "I" : "", tv->code);
+      CLog::Log(LOGDEBUG, "EGL mode %d: %s (%.2f) %s%s:%x\n", i, res.strMode, res.fPixelRatio,
+          tv->native ? "N" : "", tv->scan_mode ? "I" : "", int(tv->code));
```

Comment:
static_cast please

Also removing the .c_str() is compatible with `%s`? Or do we need `{0:s}`-style format?

Reply:
it's compatible, fmt supports both formats.

Reply:
Oh and the int() is not a cast, it creates a temporary because the type is a bitfield which fmt can't handle